tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature

```diff
@@ -58,3 +58,37 @@ Feature: checking file versions using file id
     And using new DAV path
     When user "Brian" tries to get the number of versions of file "/text.txt" using file-id path "/meta/<<FILEID>>/v"
     Then the HTTP status code should be "403"
+
+
+  Scenario Outline: check the file versions of a file after moving to a shared folder inside a project space as editor/viewer
+    Given user "Alice" has created a folder "testFolder" in space "Project1"
+    And user "Alice" has created a share inside of space "Project1" with settings:
+      | path      | testFolder |
+      | shareWith | Brian      |
+      | role      | <role>     |
+    When user "Alice" moves file "text.txt" from space "Project1" to "/testFolder/movedText.txt" inside space "Project1" using the WebDAV API
+    Then the HTTP status code should be "201"
```

Comment:
Maybe this can be the Given steps since we aren't testing the move functionality?
```suggestion
    And user "Alice" has moved file "text.txt" from space "Project1" to "/testFolder/movedText.txt" inside space "Project1" using the WebDAV API
```